scene/scene_string_names.cpp

```diff
@@ -130,6 +130,8 @@ SceneStringNames::SceneStringNames() {
 	shader_overrides_group = StaticCString::create("_shader_overrides_group_");
 	shader_overrides_group_active = StaticCString::create("_shader_overrides_group_active_");
 
+	__custom_type_script = StaticCString::create("__custom_type_script");
```

Comment:
This is *very* unorthodox syntax for this repo. Is there a functional reason for the double-underscore, or is it strictly stylistic?